sqlalchemy-stubs/sql/ddl.pyi

```diff
@@ -83,9 +103,11 @@ class CreateTable(_CreateDropBase):
     include_foreign_key_constraints: Any = ...
     def __init__(
         self,
-        element: Any,
-        bind: Optional[Any] = ...,
-        include_foreign_key_constraints: Optional[Any] = ...,
+        element: Table,
+        bind: Optional[Union[Engine, Connection]] = ...,
```

Comment:
can we create an alias for this type? it is something that it's done in stubs? like `_Bind = Optional[Union[Engine, Connection]]`